### ISSUE TITLE
Fabric 5.5 - Import table from XSD

### ISSUE THREAD

**Issue by fabiocorreak2view** [k2view-academy/K2View-Academy#49]
I need to consume a SOAP WS and then save the retrieved data into LU tables.

On Studio there is an option to import tables from XSD. It creates many tables, links them according to response schema hierarchy and define empty root functions for each table connection. It also adds PK _ID and FK _PID to join the new tables.

What is the idea/idiom behind this pattern?
How the WS response is split into or spread among the tables?
This seems to me that it would fit when parsing the XML using SAX (instead of DOM).
Do you have examples of how these root functions would look like?

Thanks.

**Comment by taliei** [comment]
The functionality of creating an LU table based on XSD file enables you to create a set of LU tables from a given XSD. Each table represents a complex element of the XML. This can be useful when you need to create the LU schema based on XSD file. Still- you need to parse the output of the WS to populate the LU tables. 
You need to build a root function to invoke the WS, parse the output and yields an array of Objects (with the type Object []).
Then you can link the output the LU table.

**Comment by neenjapt** [comment]
Hi,

Thanks for the clarification and prompt reply @taliei.

We've setup a PoC to try and enable this approach but we came across a problem regarding the JAX-WS Java objects being handled between the LU tables as I'll try to detail below.

The SOAP WS output follows this hierarchy: 
![image](https://user-images.githubusercontent.com/2249222/86943941-f7569080-c13e-11ea-94df-b7b667df926f.png)

And we tried to:
- create a table with columns for **timestamp** (String), **transactionReference** (String), **customerAccountNumber** (String) and **productPriceList** (BLOB)
- create a table with a column for each **package** field

When we run this PoC, the **productPriceList** column only holds the original Object variable identifier (in the form of _[com.zentive.minerva.webservice.meta.ProductPrices@653fb8d1]_) and its class (obtained via _getClass()_) changes to _java.lang.String_.

Is there a way to access the JAX-WS object on a population other than the one where it was returned to?

Thanks,
Bruno
